configs/components/razor-server.rb

```diff
@@ -10,38 +10,48 @@
     pkg.requires "libarchive-dev"
   end
 
-  java_home = ""
-  javacmd = ""
+  java_build_requires = ''
+  java_requires = ''
+  java_home = ''
+  service_name = 'razor-server'
   case platform.name
-  when /(el-(6|7)|fedora-(f22|f23))/
-    pkg.build_requires 'java-1.8.0-openjdk-devel'
-    pkg.requires 'java-1.8.0-openjdk'
-  when /(debian-(7|8)|ubuntu-(12|14))/
-    pkg.build_requires 'openjdk-7-jdk'
-    pkg.requires 'openjdk-7-jre-headless'
+  when /el-(6|7)/
+    java_build_requires = 'java-1.8.0-openjdk-devel'
+    java_requires = 'java-1.8.0-openjdk'
+  when /(debian-(7|8)|ubuntu-14)/
+    java_build_requires = 'openjdk-7-jdk'
+    java_requires = 'openjdk-7-jre-headless'
     java_home = "JAVA_HOME='/usr/lib/jvm/java-7-openjdk-#{platform.architecture}'"
-  when /(debian-9|ubuntu-(15|16))/
-    pkg.build_requires 'openjdk-8-jdk'
-    pkg.requires 'openjdk-8-jre-headless'
+  when /(debian-9|ubuntu-16)/
+    java_build_requires = 'openjdk-8-jdk'
+    java_requires = 'openjdk-8-jre-headless'
     java_home = "JAVA_HOME='/usr/lib/jvm/java-8-openjdk-#{platform.architecture}'"
   end
-  jruby = "#{java_home} #{javacmd} #{settings[:torquebox_prefix]}/jruby/bin/jruby -S"
+  if settings[:pe_package]
+    java_build_requires = 'pe-java'
```

Comment:
I am curious about why this doesn't need the jdk for building but the foss packages do. I'm not blocking on this, it's just something I noticed.

Reply:
honestly no idea.
but potentially something to investigate for the future.